applications/plugins/Compliant/numericalsolver/EigenSparseResponse.inl

```diff
@@ -40,15 +40,14 @@ void EigenSparseResponse<LinearSolver,symmetric>::factor(const rmat& H) {
         else m_factorized = true;
     }
 
-    if( this->f_printLog.getValue() ) serr<<"factor: "<<H<<sendl;
-
+    msg_info() << "factor: ";
```

Comment:
H is missing...thanks for pointing. 
And it cannot be the same because in the discussion related to this PR we decided that warning and error cannot be muted with the printLog. 
So it can either be: 
    msg_info() << "factor " ;  (in this case it preserve the mutability via the printLog) 
or  
    msg_warning() << "factor"; (in this case  it preserve the "Warning" level of the message)

As the message was not really informative on why it was a warning (no text describing what what wrong, how to remove the warning etc...).  I though it was more a kind of general information...and thus convert it to msg_info.